src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,8 @@ public CommandResult execute() {
         }
     }
 
-}
+    @Override
+    public boolean isMutating() {
+        return true;
+    }
+}
```

Comment:
Do not leave a "no new line character" at the end. Probably you have to change your editor settings.